docs/topics/native/native-binary-licenses.md

```diff
@@ -47,14 +47,6 @@ Always include the following license files for the corresponding projects:
         <td><a href="https://github.com/ianlancetaylor/libbacktrace">libbacktrace</a></td>
         <td><a href="https://github.com/JetBrains/kotlin/blob/master/kotlin-native/licenses/third_party/libbacktrace_LICENSE.txt">3-clause BSD license with copyright notice</a></td>
    </tr>
-   <tr>
-        <td><a href="https://github.com/microsoft/mimalloc">mimalloc</a></td>
-        <td>
-          <p><a href="https://github.com/JetBrains/kotlin/blob/master/kotlin-native/licenses/third_party/mimalloc_LICENSE.txt">MIT license</a></p>
-          <p>Include in case you use the mimaloc memory allocator instead of the default one (the <code>-Xallocator=mimalloc</code> compiler option is set).</p>
-          <p>For more information on allocators, see <a href="native-memory-manager.md">Kotlin/Native memory management</a></p>
-        </td>
-   </tr>
```

Comment:
I don't think we can remove that just yet. `mimalloc` still can be enabled by `-Xallocator=mimalloc`.

Reply:
OK, then I just remove the link for now, since we don't have info on mimaloc on the main page any more

We need not to forget to delete this when `-Xallocator=mimalloc` is completely removed